application/modules/admin/views/scripts/response/edit-tb-response.phtml

```diff
@@ -120,6 +119,7 @@ if (isset($this->responseData['shipment']["shipment_test_report_date"]) &&
         <td>
             <select class="form-control" name="assay" id="assay" class="form-control"
                     title="Please choose the assay type">
+                <option value="">--Select--</option>

```

Comment:
We are still missing this blank option in application/modules/admin/views/scripts/evaluate/edit-tb-evaluation.phtml

Reply:
file is added please review now

Reply:
No it is not. Please triple check?

Reply:
check this,  https://github.com/SystemOneId/ept/commit/42840bc0b8c2e1efb2cfdbe650b4783b60bce61e

Reply:
There is no --Select-- option on the assay field. Please add it?

Reply:
added please review now , https://github.com/SystemOneId/ept/pull/12/commits/1de558d020f1df6d4efef6d6eac3b7ac5fcdf6f2